content/en/smart-contracts/fundamentals/FAQs/index.md

```diff
@@ -131,3 +130,4 @@ It’s not impossible but storage providers are incentivized not to close the st
 #### How do I check a storage provider’s balance with their FEVM address
 
 You can query balance of any address using [Zondax's API](https://docs.zondax.ch/openapi#tag--Account).
+<!--REVIEWED!-->
```

Comment:
Note to @ElPaisano unresolved links 

  ERROR: 2 dead links found in content/en/smart-contracts/fundamentals/FAQs/index.md !
  [✖] https://kb.factor8.dev/docs/filecoin/testnets/hyperspace → Status: 0
  [✖] https://docs.zondax.ch/openapi#tag--Account → Status: 404